### PR TITLE
Implement validate and no_ack subscriber options (#926)

### LINKED ISSUE #926: Feature: new subscriber options
Add subscriber `validate: bool` option to propogate it to FastDepends [cast option](https://github.com/Lancetnik/FastDepends/releases/tag/2.2.0)

Also, add `no_ack: bool` subscriber option to all brokers to disable inner `_proccess_message` context

### PR DESCRIPTION
**Pull request by mihail8531**
# Description

Implement validate and no_ack subscriber options.

Fixes #926

## Type of change

Please delete options that are not relevant.

- [ ] Documentation (typos, code examples, or any documentation updates)
- [ ] Bug fix (a non-breaking change that resolves an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would disrupt existing functionality)
- [x] This change requires a documentation update

## Checklist

- [x] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
- [x] I have conducted a self-review of my own code
- [x] I have made the necessary changes to the documentation
- [x] My changes do not generate any new warnings
- [x] I have added tests to validate the effectiveness of my fix or the functionality of my new feature
- [x] Both new and existing unit tests pass successfully on my local environment by running `scripts/test-cov.sh`
- [x] I have ensured that static analysis tests are passing by running `scripts/static-anaylysis.sh`
- [ ] I have included code examples to illustrate the modifications

### CODE REVIEW

File: faststream/broker/core/abc.py
Comment (Lancetnik) [inline faststream/broker/core/abc.py:276]: Docstrings should ends with a dot `.` to be consistent with other docstrings

File: tests/brokers/kafka/test_consume.py
Comment (Lancetnik) [inline tests/brokers/kafka/test_consume.py:171]: Please, move this testcase from final broker case to basic TestCase class: it has no broker-based dependencies, so we can write it at once and need no create a copies

**Review by Lancetnik — CHANGES_REQUESTED** [review]
Thanks for your great job! It is really cool!

But I have a little comments alread. Also, I wish to ask you about brokers' `router.pyi` and `fastapi.pyi` signature changes

**Review by Lancetnik — DISMISSED** [review]

**Comment by Lancetnik** [comment]
Wait a bit, I need to refactor some logic before it can be merged

**Review by Lancetnik — APPROVED** [review]

**Review by Lancetnik — CHANGES_REQUESTED** [review]
Can you add these options to broker.__init__ too (apply it to all subscribers by default, by overrides it by subscriber value if it is setted)?

**Comment by davorrunje** [comment]
@mihail8531 @Lancetnik is not available this week, we are waiting for him to return to merge this PR.

**Comment by Lancetnik** [comment]
@davorrunje we are ready

**Review by davorrunje — APPROVED** [review]

**Pull request merged**
